### PR TITLE
[Semantic Browser] physical links the corresponding connected ports are not displayed within the semantic browser

### PR DESCRIPTION
**Pull request by mchoubby**
add port and owner of the port of a physical link in referenced and referencing elements within semantic browser.
add junit

### CODE REVIEW

File: core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalPortOwner.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package org.polarsys.capella.core.semantic.queries.basic.queries;
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalPortOwner.java:1]: Can you add copyright headers like other files on new files
https://github.com/eclipse/capella/wiki/How-to-contribute#license-headers

File: core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalPortOwner.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import org.polarsys.capella.common.helpers.query.IQuery;
+import org.polarsys.capella.core.data.cs.Component;
+import org.polarsys.capella.core.data.cs.PhysicalPort;
+
+/**
+ * return container of a @PhysicalPort
+ */
+public class PhysicalPortOwner implements IQuery {
+
+	@Override
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalPortOwner.java:16]: can you use the formatting of Capella 
https://github.com/eclipse/capella/wiki/How-to-contribute#code

**Review by pdulth — CHANGES_REQUESTED** [review]

File: core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalPortOwner.java (hunk 1 above)
Comment (mchoubby) [reply core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalPortOwner.java:14]: done

File: core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalPortOwner.java (hunk 2 above)
Comment (mchoubby) [reply core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalPortOwner.java]: i applied the capella formatter on eclipse
but has no effect on this file

File: core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalLinkSource.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package org.polarsys.capella.core.semantic.queries.basic.queries;
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalLinkSource.java:1]: can you add copyright header and fix formatting (on all your new files)

File: core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalLinkTarget.java [hunk 4]
```diff
@@ -0,0 +1,28 @@
+package org.polarsys.capella.core.semantic.queries.basic.queries;
```
Comment (pdulth) [inline core/plugins/org.polarsys.capella.core.semantic.queries/src/org/polarsys/capella/core/semantic/queries/basic/queries/PhysicalLinkTarget.java:1]: can you add copyright header and fix formatting

File: tests/plugins/org.polarsys.capella.test.semantic.queries.ju/src/org/polarsys/capella/test/semantic/queries/ju/testcases/PhysicalLinkSourceTest.java [hunk 5]
```diff
@@ -0,0 +1,17 @@
+package org.polarsys.capella.test.semantic.queries.ju.testcases;
```
Comment (pdulth) [inline tests/plugins/org.polarsys.capella.test.semantic.queries.ju/src/org/polarsys/capella/test/semantic/queries/ju/testcases/PhysicalLinkSourceTest.java:1]: can you add copyright header and fix formatting

File: tests/plugins/org.polarsys.capella.test.semantic.queries.ju/src/org/polarsys/capella/test/semantic/queries/ju/testcases/PhysicalLinkTargetTest.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
+package org.polarsys.capella.test.semantic.queries.ju.testcases;
```
Comment (pdulth) [inline tests/plugins/org.polarsys.capella.test.semantic.queries.ju/src/org/polarsys/capella/test/semantic/queries/ju/testcases/PhysicalLinkTargetTest.java:1]: can you add copyright header and fix formatting

**Review by pdulth — CHANGES_REQUESTED** [review]

**Comment by pdulth** [comment]
FYI https://github.com/eclipse/capella/wiki/How-to-contribute#commit-title-and-pull-request-title-format

**Pull request merged**